tests/astro_tools/test_core.py

```diff
@@ -22,11 +22,11 @@
 from rail.core.stage import RailStage
 from rail.tools.util_photometry import HyperbolicMagnitudes, HyperbolicSmoothing, PhotormetryManipulator, LSSTFluxToMagConverter, Dereddener
 from rail.core.utils import RAILDIR
-#from rail.tools.utilStages import (
+#from rail.tools.util_stages import (
```

Comment:
```suggestion
#from rail.tools.util_photometry import (
```

Reply:
Thanks.  Nice catch.   FWIW, these were already merged into main with #https://github.com/LSSTDESC/rail_astro_tools/pull/23/files.  it looks like I didn't rebase to main for this PR.